dev/cognitiveservices/data-plane/Language/analyzetext.json

```diff
@@ -2810,7 +2824,7 @@
             "InformationMetadata",
             "TemperatureMetadata",
             "CurrencyMetadata",
-            "NumberRangeMetadata",
+            "NumericRangeMetadata",
```

Comment:
Has this already GA'd? If so, this is breaking and needs to go through the breaking change board.

Reply:
Yes, this already GA'd: https://github.com/Azure/azure-rest-api-specs/pull/24768/checks?check_run_id=14957945150. Please revert. Given it's just a name change, I don't think this rises to the level of a required breaking change.

Reply:
Metadata is not GA'd it is something we introduced this preview version. I dont know why we are getting this false positive